### ISSUE TITLE
Preferred Node ande NPM versions for Hummus

### ISSUE THREAD

**Issue by cenjennifer** [galkahana/HummusJS#145]
Hi - I'm running on node v6.5.0 and npm 3.10.3. I was wondering if you can tell me what's the most suitable versions for your library. Another team member with the same versions ran into the following errors, but resolved it by upgrading the node version to 7.5. 

`Error: Module version mismatch. Expected 48, got 51`

So, just wondering if their is a preferred version for this library. Thanks!

**Comment by galkahana** [comment]
I have absoutely no idea. So whichever works is good.

**Comment by jillmolloy** [comment]
I've tried about 3/4 versions of node and I'm still getting a mismatch error similar to that listed by @cenjennifer 

`Error: Module version mismatch. Expected 50, got 51.` for v7.0.0

I've tried v7.5.0, v7.6.0, v6.9.3 also – still no luck

**Comment by galkahana** [comment]
i'll look for something.
sounds like a node-pre-gyp issue. in the meantime. use npm install hummus --build-from-source.
i think that it should sort things out.

**Comment by galkahana** [comment]
k. let's try to recreate this. i'll need operating system info. node version you already provided. and do try npm install hummus --build-from-source and let me know if it changes anything.

**Comment by cenjennifer** [comment]
@jillmolloy  - Have you tried removing all your node modules and uninstalling node and reinstalling node and then your node modules? That seem to have solved that mismatch issue.

**Comment by jillmolloy** [comment]
@cenjennifer – yes, tried removing and reinstalling

I'm actually running this in Electron, which I realized actually uses it's own version of Node, not whatever is installed on your system. So, I believe that Hummus was compiling against my system version and then complaining when I tried to run it with Electron that the versions didn't match.

I recompiled my node_modules based on the answers [here](http://stackoverflow.com/questions/39547292/node-module-version-mismatch-expected-50) using the build version of Electron that I'm running and it seems to be working correctly now.

`npm rebuild --runtime=electron --target=1.4.15 --disturl=https://atom.io/download/atom-shell --build-from-source`

So, not sure if this is actually an issue with Hummus at all, and more and environment issue, but maybe the above will be helpful to someone else.

For reference, I'm on a Mac, OS v10.12.3 and the Node version in use by Electron is v6.5.0, Electron v1.4.15

Thanks for the suggestions and the great module, @galkahana
